src/app/api/send/route.ts

```diff
@@ -0,0 +1,45 @@
+import { NextResponse } from 'next/server';
+import { Resend } from 'resend';
+import { EmailTemplate } from '@/components/email-template';
+
+const resend = new Resend(process.env.RESEND_API_KEY);
+
+export async function POST(request: Request) {
+  try {
+    const body = await request.json();
+    const { nameForm, emailForm, messageForm } = body;
+
+    if (!nameForm || !emailForm || !messageForm) {
+      return NextResponse.json(
+        { error: 'Campos obrigatórios faltando' },
+        { status: 400 }
+      );
+    }
+
+    const { error } = await resend.emails.send({
+      from: 'SPM Contact Form <onboarding@resend.dev>',
+      to: ['pedromarinho262@gmail.com'],
```

Comment:
```suggestion
      to: [process.env.EMAIL_TO_SEND],
```
Sugiro jogar esse email que está `hard coded` para uma variável de ambiente, assim fica mais fácil de mudar e manter 🤝

Reply:
Valeu pela observação meu mestre. Mandei um novo commit com a correção desse erro. 🚀🪐